jenkins/JenkinsFile-rebuild-docker-images

```diff
@@ -173,6 +173,28 @@ pipeline {
           } // post
         } // stage
 
+        stage('ci-minimal') {
+          agent { node { label 'CPU MINIMAL' } }
```

Comment:
I couldn't find a label `CPU MINIMAL` in Jenkins. Do you mean `CPU-SMALL` ?

Reply:
Yes, sorry. I meant `CPU`.